packages/sdks/snippets/angular-ssr/src/app/advanced-child/custom-tabs/custom-tabs.component.ts

```diff
@@ -15,46 +15,32 @@ import type {
   standalone: true,
   imports: [CommonModule, Blocks],
   template: `
-    <div>
-      <h2>Custom Component with editable regions</h2>
+    <ng-container *ngIf="tabList?.length">
+      <button
+        *ngFor="let tab of tabList; let i = index"
+        [class.active]="activeTab === i"
+        (click)="activeTab = i"
+      >
+        {{ tab.tabName }}
+      </button>
 
-      <div>
-        <button
-          *ngFor="let tab of tabList; let i = index"
-          [class.active]="activeTab === i"
-          (click)="activeTab = i"
-        >
-          {{ tab.tabName }}
-        </button>
-      </div>
-
-      <div *ngIf="tabList?.length">
-        <div *ngFor="let tab of tabList; let i = index">
-          <div [style.display]="activeTab === i ? 'block' : 'none'">
-            <blocks
-              [blocks]="tabList[i].children"
-              [path]="'component.options.tabList.' + i + '.children'"
-              [parent]="builderBlock.id"
-              [context]="builderContext"
-              [registeredComponents]="builderComponents"
-            ></blocks>
-          </div>
-        </div>
-      </div>
-    </div>
+      <ng-container *ngFor="let tab of tabList; let i = index">
+        <ng-container *ngIf="activeTab === i">
+          <blocks
+            [blocks]="tabList[i].blocks"
+            [path]="'component.options.tabList.' + i + '.blocks'"
```

Comment:
on the latest gen2 sdk, you can actually simplify these `path` variables and remove `component.options.` from all of them:

```suggestion
            [path]="'tabList.' + i + '.blocks'"
```

Reply:
Oh, nice. I'm making this change and will commit it now.